_site/index.html

```diff
@@ -0,0 +1,205 @@
+then follow the on-screen instructions to install Wasmtime.</p>
+<pre style="background-color:#2b303b;">
+<span style="color:#8fa1b3;">curl</span><span style="color:#c0c5ce;"> https://wasmtime.dev/install.sh</span><span style="color:#bf616a;"> -sSf </span><span style="color:#c0c5ce;">| </span><span style="color:#8fa1b3;">bash
+</span></pre>
+</div>
+<div id="platform-instructions-default" class="db">
+<p>If you’re running macOS or Linux, download and install Wasmtime by running the
+following in your terminal and following the on-screen instructions.</p>
+<pre style="background-color:#2b303b;">
+<span style="color:#8fa1b3;">curl</span><span style="color:#c0c5ce;"> https://wasmtime.dev/install.sh</span><span style="color:#bf616a;"> -sSf </span><span style="color:#c0c5ce;">| </span><span style="color:#8fa1b3;">bash
+</span></pre>
+<hr />
+<p>If you are running Windows, download and run the <a href="https://github.com/bytecodealliance/wasmtime/releases/download/dev/wasmtime-dev-x86_64-windows.msi">Wasmtime
+Installer</a>
+then follow the on-screen instructions.</p>
+<hr />
+<p>Alternatively, you can download releases for all supported platforms from the
+<a href="https://github.com/bytecodealliance/wasmtime/releases">Wasmtime GitHub page</a>.</p>
+</div>
+</div>
+</section>
+<section class="section-tinted">
+<div class="container w-container">
+<h2>Example</h2>
+<p>If you've got the <a href="https://www.rust-lang.org/tools/install">Rust compiler
+installed</a> then you can take some Rust
+source code:</p>
+<pre style="background-color:#2b303b;">
+<span style="color:#b48ead;">fn </span><span style="color:#8fa1b3;">main</span><span style="color:#c0c5ce;">() {
+</span><span style="color:#c0c5ce;">    println!(&quot;</span><span style="color:#a3be8c;">Hello, world!</span><span style="color:#c0c5ce;">&quot;);
+</span><span style="color:#c0c5ce;">}
+</span></pre>
+<p>and compile/run it with:</p>
+<pre style="background-color:#2b303b;">
+<span style="color:#8fa1b3;">$</span><span style="color:#c0c5ce;"> rustup target add wasm32-wasi
+</span><span style="color:#8fa1b3;">$</span><span style="color:#c0c5ce;"> rustc hello.rs</span><span style="color:#bf616a;"> --target</span><span style="color:#c0c5ce;"> wasm32-wasi
+</span><span style="color:#8fa1b3;">$</span><span style="color:#c0c5ce;"> wasmtime hello.wasm
+</span><span style="color:#8fa1b3;">Hello,</span><span style="color:#c0c5ce;"> world!
+</span></pre>
+</div>
+</section>
+<section>
+<div class="container w-container">
+<h2>Features</h2>
+<ul>
+<li>
+<p><strong>Fast</strong>. Wasmtime is built on the optimizing <a href="https://github.com/bytecodealliance/wasmtime/blob/main/cranelift/README.md">Cranelift</a> code generator to
+quickly generate high-quality machine code either at runtime or
+ahead-of-time. Wasmtime's runtime is also optimized for cases such as
+efficient instantiation, low-overhead transitions between the embedder and
+wasm, and scalability of concurrent instances.</p>
```

Comment:
This sentence could maybe be simplified:

```suggestion
ahead-of-time. Wasmtime is optimized for efficient
instantiation, low-overhead transitions between the embedder and
wasm, and scalability of concurrent instances.</p>
```